strings/traumas.json

```
@@ -13,59 +12,75 @@
 		"GEY AWAY FROM ME U GREIFING PRICK!!!!",
-		"ur a fuckeing autist!",
-		"@pick(semicolon)HELP SHITECIRTY MURDERIN  MEE!!!",
+		"@pick(semicolon)HELP SHITECIRTY MURDERIN MEE!!!",
 		"hwat dose tha @pick(random_gibberish) mean?????",
-		"@pick(semicolon)CAL; TEH SHUTTLE!!!!!",
+		"@pick(semicolon)DO A BLUP SPEaS JUMP!!!!!",
 		"wearnig siNGUARLTY is.... FINE haHAAA",
 		"@pick(semicolon)AI laW 22 Open door",
-		"@pick(semicolon)this SI mY stATIon......",
+		"@pick(semicolon)this SI mY sHip......",
 		"who the HELL do u thenk u r?!!!!",
 		"geT THE FUCK OUTTTT",
-		"H U G B O X",
-		"@pick(semicolon)CRASHING THIS STTAYTION WITH NIO SURVIVROS",
-		"@pick(servers) is down!! @pick(bug)",
+		"@pick(semicolon)CRASHING THIS SHIMP WITH NIO SURVIVROS",
 		"PSHOOOM",
 		"REMOVE SINGULARITY",
 		"INSTLL TEG",
 		"TURBIN IS BEST ENGIENE",
 		"SOLIRS CAN POWER THE HOLE FLEEHT ANEWAY @pick(bug)",
 		"parasteng was best",
-		"@pick(semicolon)Tajaran has warrrres, if you have coin",
 		"@pick(semicolon)I'VE GOT BALLS OF STEEL",
 		"NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
 		"i will snatch erry motherfucker birthday",
-		"@pick(semicolon)the ai and borgs are mettacomming I think",
 		"u just did the world a little bit more sad place for someone",
-		"WHERES THE SLIP REWRITE WHERE THR FUCK ID IT?",
 		"@pick(semicolon)N-NYAAAAAA~",
 		"@pick(bug)",
 		"@pick(semicolon)wtf??????????? @pick(bug)",
 		"@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
 		"DON'T EVER TUCH ME",
-		"@pick(semicolon)GIVE ME FREE ROBUX PLEASE JUST ENOUGH FOR SHIRT AND PANTS",
-		"its gonna be like 9/11 all over again but insitaead is gonna be a bitc",
-		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????"
+		"@pick(semicolon)How do I set up the. SHow do I set u p the Singu. how I the scrungulartiy????",
+		"AMOGN US IS FUNNY!!",
+		"DID YOU FUCKING.",
+		"i DEMAND!!! APOGEE-DEV BE DEOMTED!!!",
+		"@pick(semicolon)I don't ndED tEARPAHY.",
+		"@pick(semicolon)an,d CIOCK.",
+		"TUWN ME INoT A CAT!!",
+		"WHY WAES THE PEILL REMOVED???",
+		"work on the wiki please",
+		"ahelp SPAWN @pick(aspawnships) PLS",
+		"Am i allowd to kil l people if thye piss me off",
+		"IS THIS LIEK VOIDCREW???",
+		"UNiT PANICKING.",
+		"HIII!! HI!! <3 <3 <3",
+		"I LOOK LKIE BEAN,S,,, CRINGE!!!",
+		"SEET TH shIP TO kOS!!!!!",
+		"IM SynDCIATe, I ANTag, I CAN KOS......",
+		"ahelp ADMIN CAN I BE PIRTA",
+		"DOCTOR YUO DO DISCETIONS!!!",
+		"RESERCH NANIYES",
+		"REVSRSE A NY WALLS!!!",
+		"*monch",
+		"Amonger",
+		"Live mas",
+		"top ten goliath funny moments",
+		"put me... in a stew... then buryy mee...",
+		"@pick(semicolon)GTT AWYA FROM HER YOU BITHC!!",
+		"@pick(semicolon)IS TATH A FUCKIGN,, MOTH.",
+		"@pick(semicolon)BRAZIL NUMEOR UNO...!!!!",
+		"@pick(semicolon)blbue hair??? I'M GOIgN TO KILL THDT FUIKNG COW!",
+		"@pick(semicolon)HOW DO I @pick(ghetto)?????"
```

```suggestion
		"@pick(semicolon)HOW DO I @pick(ghetto)?????",
		"ough",
		"ourgh",
		"He's BALD",
		"I am normal and can be trusted with industrial chemicals",
		"Welcome to the bathroom",
		"THE AMOUNGS BROKE ALL MY BONES"
		
```
Not a single Nyaru line? For shame.

---

i wasn't around when Nyaru was popular

---

neither was I but it's like heritage at this point